hub-saml/src/test/java/uk/gov/ida/saml/hub/domain/LevelOfAssuranceTest.java

```diff
@@ -2,19 +2,18 @@
 
 import org.junit.Test;
 
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class LevelOfAssuranceTest {
 
     @Test
     public void checkOrdering() {
-        assertTrue(LevelOfAssurance.HIGH.compareTo(LevelOfAssurance.SUBSTANTIAL) > 0);
-        assertTrue(LevelOfAssurance.HIGH.compareTo(LevelOfAssurance.LOW) > 0);
-        assertTrue(LevelOfAssurance.SUBSTANTIAL.compareTo(LevelOfAssurance.LOW) > 0);
+        assertThat(LevelOfAssurance.HIGH.compareTo(LevelOfAssurance.SUBSTANTIAL) > 0);
```

Comment:
```suggestion
        assertThat(LevelOfAssurance.HIGH.compareTo(LevelOfAssurance.SUBSTANTIAL)).isGreaterThan(0);
```

Reply:
How do you feel about using this approach?  It requires less cognitive processing and is consistent with the other 'new' methods.

Reply:
Fluent assertions using 'proper' checks like this gives better error message. So if the test fails instead of seeing something like "Expected false to be true" you will see "Expected x to be greater than y"

Reply:
I swapped both level of assurances on this line and ran the test again. It did not fail. 

`assertThat(LevelOfAssurance.SUBSTANTIAL.compareTo(LevelOfAssurance.HIGH) > 0);`

`assertThat()` didn't check compareTo() result. We should use John's suggested change.

Reply:
Accepted change